portfolio/src/main/webapp/index.html

```diff
@@ -16,5 +16,7 @@ <h1>Erys Dennis' Portfolio</h1>
     </div>
 
     <p>Hi, my name is Erys Dennis. I am from Atlanta, Ga and I currently attend Fisk University!<p>
+        <p>These are my dogs Bruno and Moses playing with a bone together!<p>
+        <img src="images/playing.JPG"/>
```

Comment:
nit: remove the extra whitespace.  This should be lined up with the \<p\> tag above